addons/live-featured-project/script.js

```diff
@@ -0,0 +1,106 @@
+export default async function (addon) {
```

Comment:
Here it should be `({addon})`, not just `(addon)`. If you do this you'll have to access the real addon object like `addon.addon`.